types_test.go

```diff
@@ -64,3 +65,60 @@ func TestUnmarshalMerkleTreeLeaf(t *testing.T) {
 		}
 	}
 }
+
+func mustB64Decode(s string) []byte {
+	b, err := base64.StdEncoding.DecodeString(s)
+	if err != nil {
+		panic(err)
+	}
+	return b
+}
+
+func TestToSignedTreeHead(t *testing.T) {
+	validRootHash := mustB64Decode("cImB6R0Uh8Kp6pAatajQU8E0hYWvzbXhB79gwMHSD8A=")
```

Comment:
I have a preference for test data being in hex rather than base64, particularly if it has any kind of structure (e.g. must be exactly 32 bytes long, is the ASN1- or TLS-encoding of a structure).

Reply:
Done.